src/Microsoft.HttpRepl.Tests/Commands/ListCommandTests.cs

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.HttpRepl.Commands;
+using Microsoft.HttpRepl.Fakes;
+using Microsoft.HttpRepl.Preferences;
+using Microsoft.Repl.Parsing;
+using Xunit;
+
+namespace Microsoft.HttpRepl.Tests.Commands
+{
+    public class ListCommandTests : CommandTestsBase
```

Comment:
What about adding a positive test - one which doesn't show the warning?

Reply:
I didn't do it initially because it was a little harder to fake (the tool tries to requery the swagger url  by default, so it'll set Structure to null even if you set it to something unless you set up a response for your SwaggerEndpoint) and the integration tests already covered it. But for completeness, I went ahead and did it now.